src/Compilers/CSharp/Portable/Binder/SimpleProgramUnitBinder.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#nullable enable
+
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    /// <summary>
+    /// This binder provides a context for binding Simple Program top-level statements within a specific
+    /// compilation unit. It ensures that locals and labels are in scope, however it is not responsible
+    /// for creating the symbols. That task is actually owned by <see cref="SimpleProgramBinder"/> and
+    /// this binder simply delegates to it when appropriate. That ensures that the same set of symbols is 
+    /// shared across all compilation usints.
```

Comment:
nit: usints -> units

Reply:
> nit: usints -> units

Chuck already caught this.

---
In reply to: [384770677](https://github.com/dotnet/roslyn/pull/41926#discussion_r384770677) [](ancestors = 384770677)